src/replaykit.cs

```diff
@@ -86,5 +109,103 @@ interface RPScreenRecorderDelegate {
 		[Export ("screenRecorderDidChangeAvailability:")]
 		void DidChangeAvailability (RPScreenRecorder screenRecorder);
 	}
-}
 
+	[iOS (10,0)]
+	[BaseType (typeof (UIViewController))]
+	interface RPBroadcastActivityViewController {
+		// inlined
+		[Export ("initWithNibName:bundle:")]
+		[PostGet ("NibBundle")]
+		IntPtr Constructor ([NullAllowed] string nibName, [NullAllowed] NSBundle bundle);
+
+		[Static]
+		[Async]
+		[Export ("loadBroadcastActivityViewControllerWithHandler:")]
+		void LoadBroadcastActivityViewController (Action<RPBroadcastActivityViewController, NSError> handler);
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		IRPBroadcastActivityViewControllerDelegate Delegate { get; set; }
+	}
+
+	interface IRPBroadcastActivityViewControllerDelegate {}
+
+	[iOS (10,0)]
+	[Protocol, Model]
+	[BaseType (typeof (NSObject))]
+	interface RPBroadcastActivityViewControllerDelegate {
+		[Abstract]
+		[Export ("broadcastActivityViewController:didFinishWithBroadcastController:error:")]
+		void DidFinish (RPBroadcastActivityViewController broadcastActivityViewController, [NullAllowed] RPBroadcastController broadcastController, [NullAllowed] NSError error);
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof (NSObject))]
+	interface RPBroadcastController {
+		[Export ("broadcasting")]
+		bool Broadcasting { [Bind ("isBroadcasting")] get; }
+
+		[Export ("broadcastURL")]
+		NSUrl BroadcastUrl { get; }
+
+		[NullAllowed, Export ("broadcastControllerDelegate", ArgumentSemantic.Weak)]
+		IRPBroadcastControllerDelegate BroadcastControllerDelegate { get; set; }
+
+		[Async]
+		[Export ("startBroadcastWithHandler:")]
+		void StartBroadcast (Action<NSError> handler);
+
+		[Export ("pauseBroadcast")]
+		void PauseBroadcast ();
+
+		[Export ("resumeBroadcast")]
+		void ResumeBroadcast ();
+
+		[Async]
+		[Export ("finishBroadcastWithHandler:")]
+		void FinishBroadcast (Action<NSError> handler);
+	}
+
+	interface IRPBroadcastControllerDelegate {}
+
+	[iOS (10,0)]
+	[Protocol, Model]
+	[BaseType (typeof(NSObject))]
+	interface RPBroadcastControllerDelegate {
+		[Abstract]
+		[Export ("broadcastController:didFinishWithError:")]
+		void DidFinish (RPBroadcastController broadcastController, [NullAllowed] NSError error);
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof (NSObject))]
+	interface RPBroadcastConfiguration : NSCoding, NSSecureCoding {
+		[Export ("clipDuration")]
+		double ClipDuration { get; set; }
+
+		[NullAllowed, Export ("videoCompressionProperties", ArgumentSemantic.Strong)]
+		NSDictionary<NSNumber, INSSecureCoding> VideoCompressionProperties { get; set; }
```

Comment:
Just a side note, We might want to add a `TODO/FIXME` here just in case apple provides docs for what keys and values can be used in a future beta release and create a StrongDictionary.